### PR TITLE
[HWSKU] add Arista-7060CX-32S-T96C8 and Arista-7060CX-32S-Q24C8

### PR DESCRIPTION
**Pull request by MichelMoriniaux**
**- What I did**
Added bcm and port_config.ini files for 2 new Arista HWSKU usable as Top of Racks or spines

**- How I did it**
Interacted with vendor to validate files

**- How to verify it**
ran in lab and production

**- Description for the changelog**
create new directories for the 2 HWSKU and added bcm and port_config.ini files

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by lguohan** [comment]
retest this please

**Comment by MichelMoriniaux** [comment]
> retest this please

Hello can you give a bit more details? what type of further testing are you expecting?

**Comment by MichelMoriniaux** [comment]
> > retest this please
> 
> Hello can you give a bit more details? what type of further testing are you expecting?

ok seeing output looks like you are looking for some unit testing, I'll try to see what I can provide as this is config and not code

**Comment by prsunny** [comment]
@MichelMoriniaux , "retest this" is a trigger for the system to rerun the checks

File: device/arista/x86_64-arista_7060_cx32s/Arista-7060CX-32S-Q24C8/qos.json.j2 [hunk 1]
```diff
@@ -0,0 +1,155 @@
+{
```
Comment (lguohan) [inline device/arista/x86_64-arista_7060_cx32s/Arista-7060CX-32S-Q24C8/qos.json.j2]: {%- include 'qos_config.j2' %}

check other skus, it is simplified.

Comment (MichelMoriniaux) [reply device/arista/x86_64-arista_7060_cx32s/Arista-7060CX-32S-Q24C8/qos.json.j2]: Agreed, I have corrected the file

**Comment by MichelMoriniaux** [comment]
retest this please

**Pull request merged**